### ISSUE TITLE
alderman legislation counts should only include non-routine bills

### ISSUE THREAD

**Issue by cathydeng** [datamade/chi-councilmatic#19]
or at least split out routine vs non-routine

**Comment by fgregg** [comment]
they should be split out, I agree

**Comment by cathydeng** [comment]
so, we don't have council member legislation counts on the new councilmatic.

old chicago councilmatic has a 'top sponsors this month' on the homepage. on the council member page, it has a list of sponsored bills by topic, I think for all time.

do we want to have council member legislation counts on the new councilmatic? if we want to add it, we should define a time period for the counts.

what time period would be most meaningful? non-routine legislation introduced since the last council meeting? over the last month? in the current legislative session? all time?

**Comment by fgregg** [comment]
Hrmm.. okay. I don't think that "top sponsor of the month" is that useful. I would like some figures that explain what a legislator has been up to on their legislator profile page, but that doesn't have to happen before launch.

**Comment by derekeder** [comment]
Related to #127

**Comment by derekeder** [comment]
closing this in favor of #55
